clj/resources/lang/en.edn

```diff
@@ -577,7 +577,10 @@
    :slipstream.ui.models.parameters.launch-mutable-run?.description             :global.nouns.mutable-deployment
    :slipstream.ui.models.parameters.launch-mutable-run?.help-hint               "Enabling this option will launch a _scalable_ deployment, which will allow the `multiplicity` for each node to be changed after launch."
 
-   :slipstream.ui.models.parameters.image-target-cloud.description              :global.nouns.cloud
+   :slipstream.ui.models.parameters.orchestrator-cost.description             "Cost for orchestrator(s)"
+   :slipstream.ui.models.parameters.orchestrator-cost.help-hint               "A _scalable_ deployment prevents the orchestrator(s) from being disposed after it is complete. This cost is already included in price estimation."
```

Comment:
Suggesting to change to "_Scalable_ deployment requires orchestrator(s) for deployment and during application runtime.  This cost ..."